docs/README.md

```diff
@@ -0,0 +1,142 @@
+sync task is run it enqueues a [`SyncManagementJob`][sync_mgmt] which does one
+of these things:
+
+* creates a new `Sync`
+* kicks off copying when a `Sync` is ready for it
+* nothing
+
+When there are no in-progress `Sync` records, then [a new one is
+started][sync_start] and that enqueues a [`StartSyncJob`][sync_start_job]. This
+job will either skip if there's nothing to do or split up the data into
+manageable chucks and enqueue [`OrganizationExportJobs`][org_export_job] for
+each part.
+
+The jobs to export a batch of `Organization` records are processed
+asynchronously and that happens on it's own queue because it's very memory
+intensive. This is where we resolve and flatten our relational data using the
+aptly named [`OrganizationResolver`][org_resolver].
+
+The second thing the management job could do is to [call `copy`][sync_copy] on
+an existing `Sync` record. Copying a Sync is another way of saying that you're
+going to finish it, thus this method enqueues a
+[`FinishSyncJob`][sync_finish_job]. This job collaborates with the
+[`DataWarehouse`][data_warehouse] to truncate and copy all data from S3 to
+Redshift.
+
+[sync_mgmt]: /app/jobs/sync_management_job.rb
+[sync_start]: /app/models/sync.rb#L8
+[sync_start_job]: /app/jobs/start_sync_job.rb
+[org_export_job]: /app/jobs/organization_export_job.rb
+[org_resolver]: /app/models/organization_resolver.rb
+[sync_copy]: /app/models/sync.rb#L25
+[sync_finish_job]: /app/jobs/finish_sync_job.rb
+[data_warehouse]: /app/models/data_warehouse.rb
+
+## Resolving Organizations
+
+Yes Bearden imports and exports, but the thing that makes it special is the way
+that it models data and resolves `Organization` records. Rather than simply
+having an email column on the organizations table, for example, we have a table
+of email addresses that point to their organization. This means the actual
+`Organization` has very little on it and serves mostly as a way to relate all
+the details of an entity. Each of these details points back to the `Source` that
+created it, which means that we always know which details are most accurate.
+
+Evaluating this accuracy happens at export time, during the process of syncing
+with Redshift. Waiting until this point in the process means means that we
+delay the flattening to as late as possible and reflect the most up-to-date
+information as possible.
+
+### Rankables
+
+A module called [`Rankable`][rankable] was extracted to wrap up the common
+behavior among classes that are created by `RawInput` records. The important
+thing it provides is a common way to get the rank of a particular record. It
+uses the `versions` table to reach back to the actor that created the record and
+then from there walk back up to the `Source` that has a rank.
+
+[rankable]: app/models/rankable.rb
+
+### The Rules
+
+The resolution rules are:
+
+* use the `Rankable` with the lowest rank (1 is better than 2)
+* break rank ties with `created_at` - newer is better
```

Comment:
#minor. Maybe keep the format consistent:

... with `created_at` (newer is better)